### ISSUE TITLE
Extension is breaking reddit.com

### ISSUE THREAD

**Issue by dnestico** [joelacus/RedditEnhancer#14]
For some reason reddit stopped working today so I checked the logs and it was full of stuff related to the extension, turned it off and reddit.com works again.

**Extension on:**
<img width="1136" alt="Screenshot 2023-05-23 at 9 54 57 PM" src="https://github.com/joelacus/RedditEnhancer/assets/3034555/4720657e-a0c5-46a4-8aa6-41b1d85ab88d">

**Extension off:**
<img width="1148" alt="Screenshot 2023-05-23 at 9 56 38 PM" src="https://github.com/joelacus/RedditEnhancer/assets/3034555/d53ae6dc-3a00-4220-b48a-5513fd8406a1">

**Comment by joelacus** [comment]
That's strange. Has it happened more than once, and does it work if you re-enable the extension and refresh reddit? Also, what logs were you reading, and what is the URL when it says "page not found"?

**Comment by dnestico** [comment]
Just turned it back on and still broken for some reason.

Here is the URL it redirects too: https://www.reddit.com/undefined 

I'm using Arc Browser btw.

Also here are the console logs I was talking about:

<img width="1220" alt="Screenshot 2023-05-23 at 11 53 47 PM" src="https://github.com/joelacus/RedditEnhancer/assets/3034555/9b76b8db-790b-4524-83f3-635d8c3a82db">

<img width="1222" alt="Screenshot 2023-05-23 at 11 54 14 PM" src="https://github.com/joelacus/RedditEnhancer/assets/3034555/2acd5268-75fe-4436-a976-bc671958544b">

<img width="1220" alt="Screenshot 2023-05-23 at 11 54 27 PM" src="https://github.com/joelacus/RedditEnhancer/assets/3034555/dea53486-794a-458f-93df-ea8dd7667af9">

**Comment by Carighan** [comment]
Arc is not in the list of supported browsers, so could very well be an inherent incompatibility.

**Comment by joelacus** [comment]
I've not heard of Arc, but that isn't the problem. I figured out what was wrong and managed to replicate your issue myself. It seems you have "Default Feed Sort Option" enabled, but haven't changed the option from default (Best) to something else. This caused the option to be "undefined" when it tried to set the default feed sort option when the page loads. I made a typo which caused this to not be handled correctly, but it should be fine in the next update.

It'll take a week for Google to approve my update, but in the meantime, either disable the "Default Feed Sort Option" or change the option in the dropdown menu.
